src/EditModel/EditDataEntryForm/useData.js

```diff
@@ -0,0 +1,87 @@
+import { useState, useEffect } from 'react'
+import { getInstance as getD2 } from 'd2/lib/d2';
+
+// Load form data, operands, indicators and flags
+const loadData = async ({ modelId }) => {
```

Comment:
This was extracted from these lines: https://github.com/dhis2/maintenance-app/blob/224f2ec40242776d1e92ab5d6fdc2cc9d4989c88/src/EditModel/EditDataEntryForm.component.js#L87-L133